webpack.config.js

```diff
@@ -38,7 +38,8 @@ var config = {
     loaders: [
       { test: /\.js$/, exclude: /(node_modules)/, loader: 'babel-loader' },
       { test: /\.jsx$/, exclude: /(node_modules)/, loader: 'babel-loader' },
-      { test: /\.less$/, loader: 'style!css!less' },
+      { test: /\.module\.less$/, loader: 'style?sourceMap!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!less?sourceMap' },
```

Comment:
Yeah. I just needed some way to make sure that they only got processed by one loader chain or the other. I had to drop the !module regex on the standard less loader in order to prevent it from processing stuff already loaded by the module loader here.